daemons/schedulerd/schedulerd_ipc.c

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright 2004-2025 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU General Public License version 2
+ * or later (GPLv2+) WITHOUT ANY WARRANTY.
+ */
+
+#include <crm_internal.h>
+
+#include <crm/crm.h>
+#include <crm/common/xml.h>
+#include <pacemaker-internal.h>
+
+#include <stdbool.h>
+#include <sys/stat.h>
```

Comment:
What are `sys/stat.h`, `sys/types.h`, and `unistd.h` being used for? Comments wouldn't hurt. I see that `uid_t`, `gid_t`, and `size_t` are in `sys/types.h`.

Also I don't see `stdbool.h` being used for anything (grepped for `bool`, `true`, and `false`).